package/libnsh/libnsh.mk

```diff
@@ -0,0 +1,19 @@
+################################################################################
+#
+# libnsh
+#
+################################################################################
+
+LIBNSH_VERSION = 0.4
+LIBNSH_PKGNAME = libnsh
+LIBNSH_SOURCE = $(LIBNSH_PKGNAME)-$(LIBNSH_VERSION).tar.gz
+# This looks like a typo, but the 'v' needs to be there
+LIBNSH_SITE = https://github.com/westermo/$(LIBNSH_PKGNAME)/releases/download/v$(LIBNSH_VERSION)
+LIBNSH_INSTALL_STAGING = YES
+LIBNSH_AUTORECONF = YES
```

Comment:
We want to avoid having to run autoreconf on packages that supply their own configure script in tarballs.  Usually only needed as a workaround when we supply patches to configure.ac or a Makefile.am.  Please drop.